### ISSUE TITLE
<feature> add exclusion class pattern

### ISSUE THREAD

**Issue by sousarka** [vojtechhabarta/typescript-generator#82]
Hi,

First of all, great tool and very useful.
Have  a request though: Is it possible to have `excludeClassPattern` similar to inclusion class pattern that's currently available?

**Comment by vojtechhabarta** [comment]
@sousarka thanks for interest in this tool.
I merged `excludeClassPatterns` configuration parameter. Could you please check if it does what you are expecting?

**Comment by sousarka** [comment]
@vojtechhabarta  thanks for checking that in. I took your changes but unfortunately it didn't work.

Here's what I have as folder structure:
x.y.model
x.y.model.ab

I have classes in all packages that end with "Constant" (which basically contains bunch of "`public static final String`" variables and I don't want those classes to be generated. The tool currently tries to convert them and creates empty interfaces (which is correct).  Here's what I added in my pom.xml to exclude them. 
```
<excludedClassPatterns>
       <pattern>x.y.model.**Constant</pattern>
</excludedClassPatterns>
```

However, It worked when I added the full class name in `excludeClasses`.
